tests/cpptests/iterator_util.h

```diff
@@ -36,7 +36,7 @@ class MockIterator {
     std::optional<std::chrono::nanoseconds> sleepTime; // Sleep for this duration before returning from Read/SkipTo
 private:
     void Init() {
-      base.type = READ_ITERATOR;
+      base.type = ID_LIST_ITERATOR;
```

Comment:
Nitpick - I guess this is more correct than `READ_ITERATOR`, but consider using `MAX_ITERATOR`